src/store/messagesStore.js

```diff
@@ -504,15 +504,15 @@ const mutations = {
 			return
 		}
 
-		const messagesToRemove = messageIds.sort().reverse().slice(199)
+		const messagesToRemove = messageIds.sort((a, b) => b - a).slice(199)
 		const newFirstKnown = messagesToRemove.shift()
 
 		messagesToRemove.forEach((messageId) => {
 			Vue.delete(state.messages[token], messageId)
 		})
 
-		if (state.firstKnown[token] && messagesToRemove.includes(state.firstKnown[token])) {
-			Vue.set(state.firstKnown, token, newFirstKnown)
+		if (state.firstKnown[token] && messagesToRemove.includes(state.firstKnown[token].toString())) {
+			Vue.set(state.firstKnown, token, +newFirstKnown)
```

Comment:
I haven't changed everything, but is it safe to change the type of `state.firstKnown`?

Reply:
`state.firstKnown[token]` is number
`newFirstKnown` is string

Reply:
Yes, now it is a `number`.

But before this PR it was a `string`. And it is a global state, so I guess, it is used in many places.

Can you confirm that it is safe to change the global state here, changing the type of `state.firstKnown[token]` from `string` to `number`?

Reply:
> before this code was never reached =/

Yes, but having only changed code I can only see that it was supposed to be as as string before and now as a number. So I can either go and check all the usage of `state.firstKnown` to be sure it is actually supposed to be a number, or ask if you have id done before PR 😀